MeesGame/Gameplay/Editor/MazeGenerator.cs

```diff
@@ -102,10 +106,16 @@ public static MeesGame.TileField GenerateMaze(MeesGame.TileField tiles, int star
                     nodesToDo.RemoveAt(position);
                 }
             }
-
+            tiles.Add(new MeesGame.EndTile(), 0, 0);
             return tiles;
         }
 
+        public static TileField GenerateMazeWithRandomStart()
```

Comment:
The generator should always do this. The level should determine the start from the tileField data.